vtkext/private/module/Testing/CMakeLists.txt

```diff
@@ -17,7 +17,8 @@ endif()
 if(F3D_MODULE_EXR)
   list(APPEND test_sources 
        TestF3DEXRReader.cxx
-       TestF3DEXRReaderInvalid.cxx)
+       TestF3DEXRReaderInvalid.cxx
+       TestF3DEXRMemReader.cxx)
```

Comment:
could you add an integration test to application/testing/CMakeLists.txt ?

Reply:
ok, i am not to sure how to but will look into it